contracts/transmuter/facets/Redeemer.sol

```diff
@@ -196,12 +197,12 @@ contract Redeemer is IRedeemer {
             // For each asset, we store the actual amount of stablecoins issued based on the `newNormalizerValue`
             // (and not a normalized value)
             // We ensure to preserve the invariant `sum(collateralNewNormalizedStables) = normalizedStables`
-            uint128 newNormalizedStables = 0;
+            uint128 newNormalizedStables;
             for (uint256 i; i < collateralListLength; ++i) {
                 uint128 newCollateralNormalizedStable = ((uint256(
                     ts.collaterals[collateralListMem[i]].normalizedStables
                 ) * newNormalizerValue) / BASE_27).toUint128();
-                newNormalizedStables += newCollateralNormalizedStable;
+                newNormalizedStables = newNormalizedStables + newCollateralNormalizedStable;
```

Comment:
So for memory variables it is better to do x=x+y over x+=y? I thought the report said it was on state variable